package.json

```diff
@@ -2,8 +2,8 @@
   "name": "recommendations-impact-dashboard",
   "scripts": {
     "test": "ts-node node_modules/jasmine/bin/jasmine src/**/*.spec.ts",
-    "build": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn package appengine:run -DskipTests",
-    "deploy": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn package appengine:deploy -DskipTests",
+    "build": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn clean && mvn package appengine:run -DskipTests",
```

Comment:
this probably adds significantly latency, which could be fine... but... maybe add a specific "clean_build" script for that? Unless you truly need to clean build every time?

Reply:
It takes more time, but it is necessary to my knowledge since Java won't compile without it. All three of us were getting a bizarre exception from the compiler, which was solved by adding `mvn clean`.